Example/InstanceID/Tests/FIRInstanceIDStoreTest.m

```diff
@@ -60,13 +53,13 @@ - (BOOL)hasCheckinPlist;
 
 @interface FIRInstanceIDStoreTest : XCTestCase
 
-@property(nonatomic) FIRInstanceIDStore *instanceIDStore;
-@property(nonatomic) FIRInstanceIDBackupExcludedPlist *checkinPlist;
-@property(nonatomic) FIRInstanceIDCheckinStore *checkinStore;
-@property(nonatomic) FIRInstanceIDTokenStore *tokenStore;
-@property(nonatomic) id mockCheckinStore;
-@property(nonatomic) id mockTokenStore;
-@property(nonatomic) id mockInstanceIDStore;
+@property(strong, nonatomic) FIRInstanceIDStore *instanceIDStore;
```

Comment:
I think strong is by default?

Reply:
I got build warnings that default was assign.